### PR TITLE
Review development docs

### PR DESCRIPTION
**Pull request by vgantchev**

### CODE REVIEW

File: docs/dev_intro.md [hunk 1]
```diff
@@ -1,39 +1,27 @@
...
-This tutorial assumes previous Substrate and Rust knowledge.
+## How to Start
 
-We encourage and support the following course to obtain and learn Substrate fundamentals.
+HydraDX is powered by Substrate which is a cutting-edge blockchain framework built on Rust. Knowledge of Rust is therefore an important prerequisite for chain development. If you want to learn Rust, a good place to start is the ["Rust Book"](https://doc.rust-lang.org/stable/book/).
 
-[Acala Runtime Developer Academy](https://www.industryconnect.org/substrate-runtime-developer-academy/)
+A further requirement is basic knowledge of the Substrate framework. If you want to get up to speed quickly, we highly recommend you to subscribe to the [Acala Runtime Developer Academy](https://www.industryconnect.org/substrate-runtime-developer-academy/).
 
-To learn Rust - [The Book](https://doc.rust-lang.org/stable/book/).
```
Comment (enthusiastmartin) [inline docs/dev_intro.md:19]: Why not the rust reference ?

**Review by enthusiastmartin — APPROVED** [review]
Nice. thanks.

Looking at this - perhaps i should not attempt to write any documentation again;-)

File: docs/dev_intro.md (hunk 1 above)
Comment (vgantchev) [reply docs/dev_intro.md:19]: who needs a book to learn Rust?

jking, it's on L17

Comment (enthusiastmartin) [reply docs/dev_intro.md:19]: ah ok. i missed that,.

**Pull request merged**